interface/wx/webview.h

```diff
@@ -161,6 +161,155 @@ enum wxWebViewIE_EmulationLevel
     wxWEBVIEWIE_EMU_IE11_FORCE = 11001
 };
 
+/**
+    @class wxWebViewHandlerRequest
+
+    A class giving access to various parameters of a webview request.
+
+    @since 3.3.0
+    @library{wxwebview}
+    @category{webview}
+
+    @see wxWebViewHandler::StartRequest()
+ */
+class WXDLLIMPEXP_WEBVIEW wxWebViewHandlerRequest
+{
+public:
+    /**
+        @return The unmodified url of the request.
+    */
+    virtual wxString GetRawURI() const = 0;
+
+    /**
+        @return The url of the request. Can be modified by the backend for
+                compatibility.
+    */
+    virtual wxString GetURI() const { return GetRawURI(); }
+
+    /**
+        @return The body data of the request of @c null if nothing was posted.
+    */
+    virtual wxInputStream* GetData() const = 0;
+
+    /**
+        @return The body data of the request as a string.
```

Comment:
Might be worth to mention that the returned string is empty if the encoding is wrong?